client/app/components/compose.coffee

```diff
@@ -311,9 +312,13 @@ module.exports = Compose = React.createClass
 
         # new draft
         else
-            state = messageUtils.makeReplyMessage @props.selectedAccountLogin,
-                @props.inReplyTo, @props.action,
-                @props.settings.get('composeInHTML')
+            state = messageUtils.makeReplyMessage(
+                @props.selectedAccountLogin,
+                @props.inReplyTo,
+                @props.action,
+                @props.settings.get('composeInHTML'),
+                @props.selectedAccount.get('signature')
```

Comment:
That's typically the kind of things that will be fixed with the refactoring next week. It will depends on the component needs, I have not checked the whole component to decide what is the best option. I believe it needs to know all the accounts (to be able to switch), so the use of  `selectedAccountID` seems to be the best option.